lib/elixir/lib/option_parser.ex

```diff
@@ -64,6 +68,9 @@ defmodule OptionParser do
 
   If a switch can't be parsed, it is returned in the invalid options list.
 
+  If you want to raise an exception for all the invalid options, please use
+  `parse!/2`.
```

Comment:
I would remove this addition; in many places we have `!` bang variants of functions, but we don't mention them in the documentation for the non-bang variants. Wdyt?